### ISSUE TITLE
Rezervacija termina na daljavo

### ISSUE THREAD

**Issue by jaanos** [ul-fmf/urnik#104]
Dobro bi bilo omogočiti rezervacijo "učilnice" *Na daljavo*, da se lahko na tedenske urnike vnese nadomestne termine, ki bodo potekali na daljavo. Seveda za take rezervacije ne bi bila potrebna potrditev referata.

**Comment by jureslak** [comment]
Najlažji workaround je, da se naredi učilnico s takim imenom in se zaenkrat ignorira "potrjenost" rezervacije.

**Comment by jaanos** [comment]
Saj učilnica že obstaja, samo izbrati je ne morem za rezervacijo (tudi, če na roko popravim HTML, da se pošlje ID 248, ga zavrne).

**Comment by jureslak** [comment]
A, potem pa najbrž za to ker ni "javna". To mora nekdo od administratorjev popravit.

**Comment by matijapretnar** [comment]
Sem uredil.
<img width="239" alt="Screenshot 2020-10-09 at 08 13 04" src="https://user-images.githubusercontent.com/1502128/95549306-4eeaed80-0a07-11eb-9aee-ddea2e8a97b6.png">

**Comment by jaanos** [comment]
Hvala, sem vnesel svoje rezervacije. Vseeno pa bi bilo dobro, da se za to učilnico ne preverjajo in poročajo konflikti.

**Comment by jaanos** [comment]
Hmm, a #105 še ni deployan? Še vedno me namreč opozori na konflikte, če poskusim rezervirati *Na daljavo*. Danes mi je Anita pobrisala te rezervacije, pa sem ji razložil, da gre za prikaze na tedenskih urnikih:) Bom pa počakal s ponovnim vnosom do deploya, da ji ne bom smetil.

Mimogrede, da ne bodo ti tedenski urniki ena taka bolj skrita stvar, bi mogoče bilo dobro, da se na urniku za splošen teden vseeno pokažejo rezervacije za trenutni teden, samo mogoče nekoliko drugače (bolj prosojno?), da se te stvari razloči. Pa en checkbox bi lahko bil, da se lahko rezervacije skrije ali razkrije (najbolje kar z javascriptom, pa mogoče en cookie, da ostane zadnja nastavitev). Fino bi bilo tudi, če bi se dalo srečanja odpovedovati (npr. če ta teden vaje odpadejo, se na tedenskem urniku ne prikažejo, na splošnem tednu pa so lahko prekrižane, če je označeno prikazovanje rezervacij).

**Comment by matijapretnar** [comment]
Naj bi bil - Jure, a si ti to preveril? Za tedenski urnik pa lahko odpremo nov issue, da ne bo zmede.

**Comment by jureslak** [comment]
Hja, sem, ampak kvaliteta kode napisana v 15 minut odmora je taka kot je. Si označil v bazi, da je možno to učilnico večkrat rezervirati?

**Comment by matijapretnar** [comment]
Ja, seveda sem! Zdajle… @jaanos - a preveriš?

**Comment by jaanos** [comment]
Ja, zdaj pri rezervaciji ne zateži več zaradi konfliktov, prav tako je rezervacija takoj potrjena. Mi pa vseeno poroča konflikte pri pregledu mojih rezervacij.
